lib/transport/selenium-webdriver/options.js

```diff
@@ -102,6 +102,12 @@ module.exports = class SeleniumCapabilities {
       return this.desiredCapabilities;
     }
 
+    // Backward compatibility for 'chromeOptions'
+    this.desiredCapabilities['goog:chromeOptions'] = Object.assign(
```

Comment:
I'm not sure if it's the best idea to always send an empty options object. How about this:

```js
if (Utils.isObject(this.desiredCapabilities.chromeOptions)) {
  this.desiredCapabilities['goog:chromeOptions'] = Object.assign(this.desiredCapabilities.chromeOptions, this.desiredCapabilities['goog:chromeOptions']);
}
```

Reply:
Done.